mtprotoproxy.py

```diff
@@ -861,13 +864,82 @@ def set_instant_rst(sock):
     return False
 
 
+async def handle_proxy_protocol(reader, writer):
+    PROXY_SIGNATURE = b"\x50\x52\x4F\x58\x59\x20"
+    PROXY_TCP4 = b"TCP4"
+    PROXY_TCP6 = b"TCP6"
+    PROXY_UNKNOWN = b"UNKNOWN"
+
+    PROXY2_SIGNATURE = b"\x0d\x0a\x0d\x0a\x00\x0d\x0a\x51\x55\x49\x54\x0a"
```

Comment:
"\r\n\r\n\0\r\nQUIT\n" is a bit shorter

Reply:
guess for binary signatures it's better to follow protocol specification, there's no benefits to make it human readable here